src/lib/plugins/system-info.js

```diff
@@ -0,0 +1,517 @@
+// This library contains several chunks of code from Sebastian Hildebrandt's
+// systeminformation library, MIT
+// https://github.com/sebhildebrandt/systeminformation
```

Comment:
Can't you add the library using npm and then use import { <function you need> } from 'systeminformation'
@NickTikhonov What do you think?